dags/solutions_team/configs/tensorflow/solutionsteam_tf_nightly_supported_config.py

```diff
@@ -142,7 +147,7 @@ def get_tf_resnet_config(
           " PYTHONPATH='.' TF_USE_LEGACY_KERAS=1"
           " python3 official/vision/train.py"
           f" --tpu={tpu_name} --experiment=resnet_imagenet"
-          " --mode=train_and_eval --model_dir=/tmp/output"
+          " --mode=train_and_eval --model_dir=/tmp/"
```

Comment:
Nit: why did we change this? It would be cleaner to sequester the output into a separate folder from the rest of tmp IMO

Reply:
The plan is to forward the logs to GCS bucket but that change will be in another PR.

Reply:
Gotcha, I'm ok with this in the meantime. It's not really a big deal anyway.